tizen/src/tasks.h

```diff
@@ -0,0 +1,111 @@
+#ifndef FLUTTER_PLUGIN_WORKMANAGER_TASKS_H_
+#define FLUTTER_PLUGIN_WORKMANAGER_TASKS_H_
+
+#include <optional>
+#include <string>
+#include <variant>
+
+#include "options.h"
+
+class WorkManagerCall {
+   public:
+    WorkManagerCall(){};
+};
+
+class InitializeTask : public WorkManagerCall {
+   public:
+    InitializeTask(int32_t callBackDispathcerHandlerKey, bool isInDebugMode)
+        : callBackDispathcerHandlerKey(callBackDispathcerHandlerKey),
+          isInDebugMode(isInDebugMode){};
+
+    int32_t callBackDispathcerHandlerKey = 0;
+    bool isInDebugMode = false;
+};
+
+class RegisterTask : public WorkManagerCall {
+   public:
+    RegisterTask(bool isInDebugMode, std::string uniquename,
+                 std::string taskname, std::optional<std::string> tag,
+                 int32_t initialDelaySeconds,
+                 std::optional<Constraints> constraintsConfig,
+                 std::optional<std::string> payload)
+        : isInDebugMode(isInDebugMode),
+          uniquename(uniquename),
+          taskname(taskname),
+          tag(tag),
+          initialDelaySeconds(initialDelaySeconds),
+          constraintsConfig(constraintsConfig),
+          payload(payload){};
+
+    bool isInDebugMode;
+    std::string uniquename;
+    std::string taskname;
+    std::optional<std::string> tag;
+    int32_t initialDelaySeconds;
+    std::optional<Constraints> constraintsConfig;
+    std::optional<std::string> payload;
```

Comment:
Please follow the naming rules.
ex) `isInDebugMode` -> `is_in_debug_mode_`

Reply:
fixed it!